terminal/terminal_other.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
@joegrasse it was just a mistake on my end, I had remembered (incorrectly) that `*_unix.go` was a valid build constraint just like `_linux` or `_darwin`, etc, just a shorthand basically. I think I had just seen it before and assumed it behaved as I thought, definitely my bad.